### PR TITLE
Refactor Central Client and Central related commands

### PR DESCRIPTION
**Pull request by pramodya1994**
Fixes #<Issue Number>

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-pull.help [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 SYNOPSIS
-       ballerina pull <org-name>/<module-name>[:<version>]
+       ballerina pull <org-name>/<package-name>[:<version>]
 
 
 DESCRIPTION
-       Pull downloads the specified module from Ballerina Central
-       along with its dependencies. It then caches this module at
+       Pull downloads the specified package from Ballerina Central
+       along with its dependencies. It then caches this package at
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-pull.help:10]:
```suggestion
       along with its dependencies. It then caches this package in the
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-pull.help [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
        '.ballerina' directory in user home.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-pull.help:11]:
```suggestion
       '.ballerina' directory in the user home.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help [hunk 3]
```diff
@@ -1,37 +1,32 @@
...
-       ballerina push <module-name>
-       ballerina push (-a | --all)
+       ballerina push
 
 
 DESCRIPTION
-       Push command pushes the specified module to Ballerina Central repository or
-       if '-a' or '--all' option is specified, then push all the modules
-       in the current project.
+       Push command pushes the current project's package to Ballerina Central
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help:9]:
```suggestion
       The Push command pushes the current project's package to Ballerina Central.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help [hunk 4]
```diff
@@ -1,37 +1,32 @@
...
+       repository.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help:10]:
```suggestion
       
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help [hunk 5]
```diff
@@ -1,37 +1,32 @@
...
 
        By default, the push command builds the sources before pushing
-       the module to Ballerina Central.
+       the package to Ballerina Central.
 
-       Before pushing modules to Ballerina Central:
+       Before pushing package to Ballerina Central:
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help:15]:
```suggestion
       Before pushing the package to Ballerina Central:
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help [hunk 6]
```diff
@@ -1,37 +1,32 @@
...
        1) Create an organization by registering with Ballerina Central at
           https://central.ballerina.io/.
        2) You will receive an access token with the registration, add it to
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help:18]:
```suggestion
       2) Add the access token with the registration you receive to
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help [hunk 7]
```diff
@@ -1,37 +1,32 @@
...
           the 'Settings.toml' in home repository (<USER_HOME>/.ballerina/).
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help:19]:
```suggestion
          the 'Settings.toml' file in the home repository (<USER_HOME>/.ballerina/).
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help [hunk 8]
```diff
@@ -1,37 +1,32 @@
...
        3) Add the created organization name and the version to the
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help:20]:
```suggestion
       3) Add the created organization name and version to the
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help [hunk 9]
```diff
@@ -1,12 +1,12 @@
 NAME
-       ballerina-search - Search Ballerina Central for modules
+       ballerina-search - Search Ballerina Central for packages
 
 SYNOPSIS
        ballerina search <keyword>
 
 
 DESCRIPTION
        Search command searches Ballerina Central for the given keyword appearing in
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help:9]:
```suggestion
       The search command searches Ballerina Central for the given keyword appearing in
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help [hunk 10]
```diff
@@ -1,12 +1,12 @@
...
-       organization name, module name, or module descriptions.
+       organization name, package name, or package descriptions.
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help:10]:
```suggestion
       the organization name, package name, or package descriptions.
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-pull.help [hunk 11]
```diff
@@ -1,18 +1,18 @@
...
 
 SYNOPSIS
-       ballerina pull <org-name>/<module-name>[:<version>]
+       ballerina pull <org-name>/<package-name>[:<version>]
 
 
 DESCRIPTION
-       Pull downloads the specified module from Ballerina Central
-       along with its dependencies. It then caches this module at
+       Pull downloads the specified package from Ballerina Central
```
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-pull.help:9]:
```suggestion
       The pull command downloads the specified package from Ballerina Central
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help (hunk 9 above)
Comment (praneesha) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help:9]:
```suggestion
       The search command searches Ballerina Central for the given keyword appearing in
```

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-pull.help (hunk 1 above)
Comment (pramodya1994) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-pull.help]: Added

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-pull.help (hunk 2 above)
Comment (pramodya1994) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-pull.help]: Added

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help (hunk 3 above)
Comment (pramodya1994) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help]: Added

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help (hunk 4 above)
Comment (pramodya1994) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help]: Added

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help (hunk 5 above)
Comment (pramodya1994) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help]: Added

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help (hunk 6 above)
Comment (pramodya1994) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help]: Added

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help (hunk 7 above)
Comment (pramodya1994) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help]: Added

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help (hunk 8 above)
Comment (pramodya1994) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-push.help]: Added

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help (hunk 9 above)
Comment (pramodya1994) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help]: Added

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help (hunk 10 above)
Comment (pramodya1994) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help]: Added

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-pull.help (hunk 11 above)
Comment (pramodya1994) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-pull.help]: Added

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help (hunk 9 above)
Comment (pramodya1994) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-search.help]: Added

**Review by praneesha — APPROVED** [review]

**Review by hevayo — APPROVED** [review]

**Pull request merged**